src/tests/Quote.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Quote from '../components/Quote';
+
+it('Quote component should render correctly', () => {
+  const tree = renderer
+    .create(<Quote />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- You have done great so far :+1: :+1: But there are a few requirements you still need to meet before moving to the next milestone. :smile: :wave: 
> Create unit tests for all React components:
Use Jest snapshots to test the components.
Use React Testing Library to simulate user interaction.

- [ ] As stated in the project requirement, you need to write tests for all the components in your project. Currently, you only wrote test for the `Quote.js` component leaving out the other components `Navbar, Home, Calculator`. I'll suggest you use Jest Snapshot to test these components too. :+1: :small_airplane: 
- [ ] I'll also suggest you remove the `App.test.js` test file as it's a boilerplate test code and it currently fails when you run the test suites. :smile: :wave: